packages/client/hmi-client/src/components/pyciemss/tera-pyciemss-cancel-button.vue

```diff
@@ -12,10 +12,14 @@ const props = defineProps<{
 	simulationRunId?: string | string[];
 }>();
 
-const disabled = computed(() => props.simulationRunId === '');
+const isCancelling = ref(false);
+const disabled = computed(
+	() => props.simulationRunId === '' || props.simulationRunId?.length === 0 || isCancelling.value
```

Comment:
We've been using Lodash `isEmpty` extensively in the application.

```suggestion
	() => isEmtpy(props.simulationRunId) || isCancelling.value
```

Reply:
typo in `isEmpty`

Reply:
Very good point thank you